tools/dawgrun/README.md

```
@@ -35,19 +35,13 @@ connection.
 
 ## Building
 
-From a `DAWGS` checkout:
+From a `DAWGS` checkout, run the tool directly:
 
     go tool dawgrun
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)